.github/ISSUE_TEMPLATE/2-feature-request.md

```diff
@@ -0,0 +1,26 @@
+---
+name: "\U0001F680Feature request"
+about: Suggest a feature for webhint
+
+---
+<!--
+
+Hi there 👋
+
+Before opening a new issue please make sure to search in the existing
+ones (even closed!) as they might contain information about workarounds,
+resolution, or progress updates.
+
+-->
+
+# 🚀 Feature request
+
+## Description
+
+<!-- ✍️ --> What do you want to be added?
+
+<!-- 💪 --> Are you willing to work on this?
+
+## What scenarios will this solve?
```

Comment:
I'd love if we put this first. Thinking about and articulating the problem more clearly often results in a better, more articulated solution.

Reply:
Asking for scenarios even after the description should help encourage this thought process for whoever files the request. I think leading with the description will help the rest of us grok the request more quickly after it gets filed.